### PR TITLE
[WFLY-10894] Tests for propagating undertow authentication mode to El…

### PR DESCRIPTION
**Pull request by spyrkob**
…ytron

JIRA: https://issues.jboss.org/browse/WFLY-10894

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:
- [ ] Pull Request title is properly formatted: `[WFLY-XYZ] Subject` or `WFLY-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue(s)
- [ ] Pull Request contains description of the issue(s)
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted

For bigger changes, major and minor component upgrades make sure your PR also meets following requirements:
- [ ] Pull Request requires a change to the documentation
- [ ] Documentation have been updated accordingly
- [ ] Tests were added to cover changes

For new features ensure as well:
- [ ] Analysis was done
- [ ] Test Plan has been done
- [ ] Tests were verified in advance

If you are not an active contributor of the WildFly project you can request sponsorship by one of the members to help guide you through the process.

### CODE REVIEW

File: testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/authmode/ConstraintDrivenAuthModeTestCase.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                }
+
+                @Override
+                public void create(CLIWrapper cli) throws Exception {
+                    cli.sendLine("/subsystem=undertow/servlet-container=default:write-attribute(name=proactive-authentication, value=false)");
+                }
+
+                @Override
+                public void remove(CLIWrapper cli) throws Exception {
+                    cli.sendLine("/subsystem=undertow/servlet-container=default:remove-attribute(name=proactive-authentication)");
```
Comment (bstansberry) [inline testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/authmode/ConstraintDrivenAuthModeTestCase.java]: There's no 'remove-attribute' op. Did you mean 'undefine-attribute'?

File: testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/authmode/ProactiveAuthModeTestCase.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                }
+
+                @Override
+                public void create(CLIWrapper cli) throws Exception {
+                    cli.sendLine("/subsystem=undertow/servlet-container=default:write-attribute(name=proactive-authentication, value=true)");
+                }
+
+                @Override
+                public void remove(CLIWrapper cli) throws Exception {
+                    cli.sendLine("/subsystem=undertow/servlet-container=default:remove-attribute(name=proactive-authentication)");
```
Comment (bstansberry) [inline testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/authmode/ProactiveAuthModeTestCase.java]: undefine-attribute

**Review by bstansberry — CHANGES_REQUESTED** [review]

File: testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/authmode/ConstraintDrivenAuthModeTestCase.java (hunk 1 above)
Comment (spyrkob) [reply testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/authmode/ConstraintDrivenAuthModeTestCase.java]: Yes sorry. Updated the PR now

**Comment by kabir** [comment]
Retest this please

**Pull request merged**